inc/class-shortcake-bakery.php

```diff
@@ -72,7 +72,8 @@ public function action_init_register_shortcodes() {
 			$this->registered_shortcodes[ $shortcode_tag ] = $class;
 			add_shortcode( $shortcode_tag, array( $this, 'do_shortcode_callback' ) );
 			$class::setup_actions();
-			$ui_args = $class::get_shortcode_ui_args();
+			$ui_args = apply_filters( 'shortcake_bakery_global_ui_args', $class::get_shortcode_ui_args() );
+			$ui_args = apply_filters( "shortcake_bakery_{$shortcode_tag}_ui_args", $ui_args );
```

Comment:
Hm. Is it really necessary to have two filters? Couldn't we just have one, and make the `$shortcode_tag` the second argument?